clang/lib/DPCT/DPCT.cpp

```diff
@@ -785,6 +785,10 @@ int runDPCT(int argc, const char **argv) {
           else if (Option.ends_with("occupancy-calculation"))
             Experimentals.addValue(
                 ExperimentalFeatures::Exp_OccupancyCalculation);
+          else if (Option.ends_with("free-function-queries"))
```

Comment:
Please split the function change into separate PR.

Reply:
Submitted #1240  to support 2 options